SwiftCompilerSources/Sources/SIL/FunctionConvention.swift

```diff
@@ -228,6 +233,24 @@ extension FunctionConvention {
   }
 }
 
+extension FunctionConvention {
+  public struct LifetimeDependencies : Collection {
+    let bridged: BridgedLifetimeDependenceInfoArray
+
+    public var startIndex: Int { 0 }
+
+    public var endIndex: Int { bridged.count() }
+
+    public func index(after index: Int) -> Int {
+      return index + 1
+    }
+
+    public subscript(_ index: Int) -> BridgedLifetimeDependenceInfo {
```

Comment:
SIL APIs shouldn't expose bridged data types. Please either define a swift struct around `BridgedLifetimeDependenceInfo` or - if you only use `LifetimeDependencies` internally in function conventions, don't make it public.

Reply:
Done